examples/translator/example.toml

```diff
@@ -0,0 +1,11 @@
+name = "translator"
+
+[application]
+manifest = "examples/translator/config/config.toml"
+out = "examples/translator/bin/config.bin"
+
+[modules]
+module = { Cargo = { cargo_manifest = "examples/translator/module/rust/Cargo.toml" } }
+
+[clients]
+cpp = { Bazel = { bazel_target = "//examples/translator/client:client" } }
```

Comment:
&hellip;specifically here.  There is no C++ in this example.

Reply:
Ahh good catch! I totally missed this, renamed it now.